build.sbt

```diff
@@ -91,6 +91,21 @@ test in assembly := {} // Should there be tests?
 
 assemblyOutputPath in assembly := file("./utils/bin/firrtl.jar")
 
+Project.inConfig(Test)(baseAssemblySettings)
+
+test in (Test, assembly) := {} // Ditto above
+
+assemblyMergeStrategy in (Test, assembly) := {
+  case PathList("firrtlTests", xs @ _*) => MergeStrategy.discard
+  case x =>
+    val oldStrategy = (assemblyMergeStrategy in (Test, assembly)).value
+    oldStrategy(x)
+}
+
+assemblyJarName in (Test, assembly) := s"firrtl-test.jar"
+
+assemblyOutputPath in (Test, assembly) := file("./utils/bin/" + (Test / assembly / assemblyJarName).value)
+
```

Comment:
Just stripping the package `firrtlTests` still includes other tests: `https://github.com/freechipsproject/firrtl/tree/master/src/test/scala/firrtl`

I'm okay with this approach since the utility of this outweighs blocking on a perfect solution, but I hope we can eventually make this a multi-project build such that we can also publish the `testutils` package to maven.